index.html

```diff
@@ -1,348 +1,328 @@
-                            <!-- <img src="" alt="place image here" /> -->
-                        </div>
-                        <div class="item-description">
-                            <div class="item-title">
-                                <h2>Mult-Post Stories</h2>
-                                <h3>Gain + Glory</h3>
-                            </div>
-                            <ul class="technologies">
-                                <li>Javascript</li>
-                                <li>Ruby on rails</li>
-                                <li>HTML</li>
-                                <li>CSS</li>
-                            </ul>
-                        </div>
-                        <div class="green-btn">
-                            <a href="single-project.html">See Project</a>
-                        </div>
+                <div class="social-icons">
+                    <ul>
+                        <li>
+                            <a href="https://github.com/Bornittah">
+                                <img src="assets/icons/github-brands.svg" class="icon" alt="icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/agasha-bornittah-47b594131/">
+                                <img src="assets/icons/linkedin-brands.svg" class="icon" alt="icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="assets/icons/angellist-brands.svg" class="icon" alt="icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/AgashaBornittah">
+                                <img src="assets/icons/twitter-brands.svg" class="icon" alt="icon">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://medium.com/@bornittah">
+                                <img src="assets/icons/medium-brands.svg" class="icon" alt="icon">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="down-arrow">
+                    <a href="">
+                        <img src="assets/icons/angle-down-solid.svg" class="icon" alt="icon">
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="section">
+            <div class="title">
+                <h2 class="primary-text">My Recent Works</h2>
+                <hr class="line" />
+            </div>
+            <div class="grid-container">
+                <div class="grid-item">
+                    <div class="item-image">
+                        <!-- <img src="" alt="place image here" /> -->
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean. Kindly remove all unused code this will make your code mush noise, It is always a good practice to keep your code clean from such comments.